resources/views/themes/default1/order/index.blade.php

```diff
@@ -220,8 +220,10 @@
         $('#order-table').DataTable({
```

Comment:
Reset button should reset all the values to before

Reply:
didn't get you. Please elaborte